openwisp_radius/models.py

```diff
@@ -85,7 +84,3 @@ def _get_instance(self, **kwargs):
 
     class Meta(AbstractRadiusUserProfile.Meta):
         abstract = False
-        swappable = swappable_setting('openwisp_radius', 'RadiusUserProfile')
```

Comment:
This line was not supposed to be deleted @Mah1ndra

Reply:
okay @rohithasrk . i'll update accordingly :+1: